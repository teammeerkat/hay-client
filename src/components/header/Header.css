.Header {
  padding: 1em;
  background-color: yellow;
  text-align: center;
  justify-content: space-around;
  flex-direction: row-reverse;

  img {
    width: 10em;
    height: 10em;
  }

  a {
    align-self: center;
  }
}
