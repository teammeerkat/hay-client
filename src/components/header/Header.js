import React from 'react';
import { Link } from 'react-router-dom';
import styles from './Header.css';
import logo from '../../../assets/Timon-KH.jpg';

export default function Header() {
  return (
    <header className={styles.Header}>
      <Link to="/">
        <img src={logo} alt="logo"/>
      </Link>
    </header>
  );
}
