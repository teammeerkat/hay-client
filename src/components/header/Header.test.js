import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header tests', () => {
  it('matches a snapshot', () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });
});
