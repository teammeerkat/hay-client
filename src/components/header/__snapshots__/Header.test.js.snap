// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests matches a snapshot 1`] = `ShallowWrapper {}`;
